test/GithubApi.Repositories.test.js

```diff
@@ -0,0 +1,106 @@
+const axios = require('axios');
+const chai = require('chai');
+const chaiSubset = require('chai-subset');
+const { expect } = require('chai');
+const { StatusCodes } = require('http-status-codes');
+const md5 = require('md5');
+
+chai.use(chaiSubset);
+
+const urlBase = 'https://api.github.com';
+const usersResource = 'users';
+const reposResource = 'repos';
+const githubUserName = 'aperdomob';
+const repositoryName = 'jasmine-json-report';
+
+const repositoryUrl = `${urlBase}/${reposResource}/${githubUserName}/${repositoryName}`;
+const userUrl = `${urlBase}/${usersResource}/${githubUserName}`;
+
+describe('Consume GET methods', () => {
+  it(`Checking name, company, and location to the given user "${githubUserName}"`, async () => {
+    const response = await axios.get(
+      `${urlBase}/${usersResource}/${githubUserName}`,
+      {
+        headers: {
+          Authorization: `${process.env.ACCESS_TOKEN}`
+        }
+      }
+    );
+
+    expect(response.status).to.equal(StatusCodes.OK);
+    expect(response.data.name).to.equal('Alejandro Perdomo');
+    expect(response.data.company).to.equal('Perficient Latam');
+    expect(response.data.location).to.equal('Colombia');
+  });
+
+  it(`Check full name, private, and description given repository name "${repositoryName}"`, async () => {
+    const response = await axios.get(`${userUrl}/repos`, {
+      headers: {
+        Authorization: `${process.env.ACCESS_TOKEN}`
+      }
+    });
+
+    expect(response.status).to.equal(StatusCodes.OK);
+
+    const repository = await response.data.find(
+      ({ name }) => name === repositoryName
+    );
+
+    expect(repository.full_name).to.equal('aperdomob/jasmine-json-report');
+    expect(repository.private).to.be.false;
+    expect(repository.description).to.equal('A Simple Jasmine JSON Report');
+  });
+
+  it('Should download repository in zip', async () => {
+    const response = await axios.get(`${repositoryUrl}/zipball/master`, {
+      headers: {
+        Authorization: `${process.env.ACCESS_TOKEN}`
+      }
+    });
+
+    expect(response.status).to.equal(StatusCodes.OK);
+    expect(response.headers['content-type']).to.equal('application/zip');
+  });
+
+  it('Getting and verifying name, path and sha from README.md', async () => {
+    const fileName = 'README.md';
+    const filePath = 'README.md';
+    const fileSha = '360eee6c223cee31e2a59632a2bb9e710a52cdc0';
+
+    const expectedFormat = {
+      name: fileName,
+      path: filePath,
+      sha: fileSha
+    };
+
+    const response = await axios.get(`${repositoryUrl}/contents`, {
```

Comment:
the repository response of the previous test can be used to get this URL (if you use `repository.url` is equivalent to the `repositoryUrl` variable):
![image](https://user-images.githubusercontent.com/10209180/176711611-92f3d2a4-def3-44be-8bbf-837f36507eb0.png)